pkgs/applications/video/glaxnimate/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, cmake
+, zlib
+, potrace
+, ffmpeg
+, libarchive
+, python3
+, qtbase
+, qttools
+, wrapQtAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "glaxnimate";
+  version = "0.5.1";
+
+  src = fetchFromGitLab {
+    owner = "mattbas";
+    repo = "${pname}";
+    rev = "${version}";
+    sha256 = "G4ykcOvXXnVIQZUYpRIrALtDSsGqxMvDtcmobjjtlKw=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    wrapQtAppsHook
+  ];
+  buildInputs = [
+    zlib
+    potrace
+    ffmpeg
+    # gst_all_1.gst-libav # Note that since gst-libav-1.6, libav is actually ffmpeg
+    libarchive
+    python3
+    qtbase
+    qttools
+  ];
+
+  meta = with lib; {
+    homepage = "https://gitlab.com/mattbas/glaxnimate";
+    description = "Simple vector animation program.";
+    license = licenses.gpl3;
+    maintainers = with maintainers; [ tobiasBora ];
+    platforms = platforms.linux;
```

Comment:
```suggestion

```
It also builds on aarch64-darwin, so you can remove that line which means it works on the default platforms.

Reply:
where is default platforms list ? freebsd is inside ?

Reply:
@Et7f3 Oh, you really made my day!